java-examples/src/main/java/io/yash/jdbc/SqlTest.java

```diff
@@ -0,0 +1,45 @@
+package io.yash.jdbc;
+
+import java.sql.*;
+//import com.mysql.jdbc.Connection;
+
+public class SqlTest {
+	public static void main(String[] args) throws ClassNotFoundException, SQLException {
+		
+		
+		
+		String uname = "root";
```

Comment:
move to static constants. create a connection through Junit and keep the logic and code here.
Then add tests for exceptions raised in case of wrong password, invalid query and everything.